client/app/account/billing/autoRenew/billing-autoRenew.controller.js

```diff
@@ -5,6 +5,7 @@
  * Autorenew services configuration
  */
 angular.module("Billing.controllers").controller("Billing.controllers.AutoRenew", [
+    "$rootScope",
```

Comment:
I won't refactor lines that I haven't touched on this PR, no need to ask

Reply:
What's the reason of such a comment?